src/main/java/seedu/address/model/AddressBook.java

```diff
@@ -18,7 +18,7 @@
  */
 public class AddressBook implements ReadOnlyAddressBook {
 
-    private final UniquePersonList persons;
+    private final UniqueApplicantList persons;
```

Comment:
Time to change the variable name too for consistency

Reply:
Agreed. Thanks for pointing it out will work on resolving this